scripts/build/bundler.js

```diff
@@ -118,6 +147,9 @@ function getRollupConfig(bundle) {
 
   const babelConfig = getBabelConfig(bundle);
 
+  const alias = { ...bundle.alias };
+  alias.entries = [...entries, ...(alias.entries || [])];
```

Comment:
Does this work?

```suggestion
  const alias = { ...bundle.alias, entries: [...entries, ...(bundle.entries || [])] };
```

Reply:
Yes, I think so.

Reply:
Wait, no... bundle.alias can be undefined.